build.gradle.kts

```diff
@@ -24,6 +23,7 @@ subprojects {
     }
 
     repositories {
+        mavenLocal()
```

Comment:
Is this supposed to be committed?

Reply:
Unfortunately, ya. Until we can get contracts bootstrapped in test you'll always have to run contracts from your local maven repo.

Reply:
If that's the case, shouldn't `Dependencies.Provenance.LoanPackage` be removed?

Reply:
Ya the other part to that is that all the contracts you want to run have to be in the classpath for discovery via reflection. It's a bit of a pain tbh.